docs/testing/playwright.mdx

```diff
@@ -273,4 +273,101 @@ test('ensure that clerk has loaded', async ({ page }) => {
 })
 ```
 
+## Testing authenticated flows
+
+Playwright executes tests in isolated environments called browser contexts. So, each test case runs in a new browser context, which means that the user session is not shared between test cases. However, tests can load existing authenticated state in Playwright.
+
+Following the [Playwright docs on Authentication](https://playwright.dev/docs/auth), you can test authenticated flows without having to sign in for each test case.
+
+The steps below showcase an example of how to save the session storage in a file and load it in your test cases. Of course, you can customize this to fit your needs. You can also have a look at our example repository [here](https://github.com/clerk/clerk-playwright-nextjs).
+
+<Steps>
+  ### Add the storage file in your `.gitingore`
+
+  Create a `playwright/.clerk` directory and add it to your `.gitignore`.
+
+  ```sh {{ filename: 'terminal' }}
+  mkdir -p playwright/.clerk
+  echo $'\nplaywright/.clerk' >> .gitignore
+  ```
+
+  ### Save the session in your setup file
+
+  Save the session in your global setup file. This file will be executed before all projects.
```

Comment:
```suggestion
  Save the session in your [global setup file](https://playwright.dev/docs/test-global-setup-teardown). This file is executed before all projects.
```

Reply:
Is this so you only have to sign in once to be authenticated for all of your tests?

Reply:
Yes, this is what this new section showcases: "How to sign in once and re-use the signed-in state to many tests (all of your tests, or some of your tests, whatever the user wants)".